feat-content.html

```diff
@@ -2,9 +2,9 @@
 
     <div class="feat-content-card">
         <a class="cardClick" target="_blank" rel="noopener noreferrer"
-           href="https://bcc2020.github.io/">
-            <h4 class="feat-content-title">Bioinformatics Community Conference 2020</h4>
-            <p class="feat-card-description">Registration for BCC2020 is now open! Check out the full program and sign up for Dockstore's virtual workshops on Docker, Descriptors, and analysis with Terra.</p>
+           href="https://docs.dockstore.org/en/develop/getting-started/github-apps/github-apps-landing-page.html">
+            <h4 class="feat-content-title">Dockstore GitHub App for Continuous Integration</h4>
```

Comment:
I don't think I like "Continuous Integration" -- for me it implies a build server. I would just say "Dockstore GitHub App".